tutorials/v7/box.cxx

```diff
@@ -26,15 +26,12 @@ void box()
    auto canvas = RCanvas::Create("RBox drawing");
 
    auto box1 = canvas->Draw<RBox>(RPadPos(0.1_normal, 0.3_normal), RPadPos(0.3_normal,0.6_normal));
-   RColor Color1(0, 255, 0, 0.5); // 50% opaque
-   RColor Color2(0, 0, 255, 0.7); // 70% opaque
-
-   box1->AttrBorder().SetColor(Color1).SetWidth(5);
-   box1->AttrFill().SetColor(RColor::kRed);
+   box1->AttrBorder().SetColor(RColor::kBlue).SetWidth(5);
+   box1->AttrFill().SetColor(RColor(0, 255, 0, 127)); // 50% opaque
```

Comment:
It would be good to keep the alpha-float overload, that's really more useful than a 255-based one... I'd rather see the RGB part be float than everything becoming char.

Reply:
Problem, that alpha parameter stored as `uint8_t` and setting/getting it as float may confuse users.
```
root [6] col.SetAlphaFloat(0.5)
root [7] col.GetAlphaFloat()
(float) 0.498039f
root [8] 
```

Reply:
Well, in principle we could just as well use float values internally, as per https://developer.mozilla.org/en-US/docs/Web/CSS/alpha-value - using the "percentage" notation.

Reply:
Yes, but then it should be special type. 
And default value should be 0..255

Reply:
I disagree :-)

I think we need to state arguments for our opinions to make progress:

To me, it's more user-friendly to think in "percent opacity" than to estimate what "194/255" is. So for me, the default user interface should take floating point opacity.

This is so important that I'd rather chance RGB to also be floating point, for consistency reason, than to have opacity char for consistency.

The argument that the internal representation is a char is an implementation detail: we might as well store it as float. Same for the RGB values, by the way. If the class size matters then fine, char it is - but I don't expect hoards of `RColor`s like we had with `TColor`, as nowadays these things get sampled from palettes, and we don't pre-allocate a large array of colors.

What am I missing?

Reply:
Would it be consequent to use `0..1` range for all colors components - `R`, `G`, `B` and `A`?

Reply:
Yes, sorry, that's what I implicitly meant with

> I'd rather change RGB to also be floating point